airbyte-scheduler/scheduler-persistence/src/main/java/io/airbyte/scheduler/persistence/job_error_reporter/SentryExceptionHelper.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.scheduler.persistence.job_error_reporter;
+
+import io.airbyte.commons.lang.Exceptions;
+import io.sentry.protocol.SentryException;
+import io.sentry.protocol.SentryStackFrame;
+import io.sentry.protocol.SentryStackTrace;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class SentryExceptionHelper {
+
+  /**
+   * Processes a raw stacktrace string into structured SentryExceptions
+   * <p>
+   * Currently, Java and Python stacktraces are supported. If an unsupported stacktrace format is
+   * encountered, `null` will be returned, in which case we can fall back to message-based grouping.
```

Comment:
Couldn't the fallback be sending the stacktrace as a string like we tested? It's not perfect but surely better than null and relying on internal message only?